### PR TITLE
Added "add new cube" option to your cubes dropdown

### LINKED ISSUE #1903: Add "Create A New Cube" to "Your Cubes" dropdown
**What would you like to be able to do?**

It would be nice to add an option to create a cube to the "Your Cubes" dropdown, in addition to (or instead of?) the user profile dropdown. 

Creating cubes feels like it belongs alongside my existing cubes.

Not a major issue since creating an additional cube is pretty rare.

### PR DESCRIPTION
**Pull request by dekkerglen**
Fixes #1903 

image
image

### CODE REVIEW

**Comment by lunakv** [comment]
Visually, I'm not a fan of the bold text. It stands out too much. The rule above the link is imo enough to separate it.

File: src/layouts/MainLayout.js [hunk 1]
```diff
@@ -110,11 +110,17 @@ const MainLayout = ({ user, children, loginCallback }) => {
                           Your Cubes
                         </DropdownToggle>
                         <DropdownMenu right>
-                          {user.cubes.map((item) => (
-                            <DropdownItem key={`dropdown_cube_${item.name}`} href={`/cube/overview/${item._id}`}>
-                              {item.name}
-                            </DropdownItem>
-                          ))}
+                          <>
```
Comment (lunakv) [inline src/layouts/MainLayout.js:113]: Is there a reason this fragment needs to be here?

Comment (dekkerglen) [reply src/layouts/MainLayout.js:113]: It's need, a root element can only contain one jsx tag

Comment (dekkerglen) [reply src/layouts/MainLayout.js:113]: NVM: was looking at the wrong one, this can be removed

**Pull request merged**